widgets/hits.js

```diff
@@ -2,7 +2,12 @@ var React = require('react');
 
 var utils = require('../lib/utils.js');
 
-function hits({container = null, templates = {}, hitsPerPage = 20}) {
+function hits({
+    container = null,
+    templates = {},
+    transformData = {},
```

Comment:
> What about changing the name when it's not a function but an object? transformers maybe?

I am ok with keeping transformData even when it's an object, if documented that's ok.